<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RGroupPopup</class>
 <widget class="QWidget" name="RGroupPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QToolButton" name="new_rgroup_btn">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
}</string>
     </property>
     <property name="text">
      <string>R+</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="rgroup1_btn">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
}</string>
     </property>
     <property name="text">
      <string>R1</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="rgroup5_btn">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
}</string>
     </property>
     <property name="text">
      <string>R5</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="rgroup6_btn">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
}</string>
     </property>
     <property name="text">
      <string>R6</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="rgroup7_btn">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
}</string>
     </property>
     <property name="text">
      <string>R7</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="rgroup2_btn">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
}</string>
     </property>
     <property name="text">
      <string>R2</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="rgroup3_btn">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
}</string>
     </property>
     <property name="text">
      <string>R3</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QToolButton" name="rgroup8_btn">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
}</string>
     </property>
     <property name="text">
      <string>R8</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QToolButton" name="rgroup9_btn">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
}</string>
     </property>
     <property name="text">
      <string>R9</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QToolButton" name="rgroup4_btn">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
border : none;
font: bold 14px;
}</string>
     </property>
     <property name="text">
      <string>R4</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="sketcher.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="group"/>
 </buttongroups>
</ui>
